x-pack/plugins/fleet/server/services/epm/elasticsearch/template/install.ts

```diff
@@ -85,30 +85,11 @@ const installPreBuiltTemplates = async (paths: string[], esClient: Elasticsearch
     const { file } = getPathParts(path);
     const templateName = file.substr(0, file.lastIndexOf('.'));
     const content = JSON.parse(getAsset(path).toString('utf8'));
-    let templateAPIPath = '_template';
 
-    // v2 index templates need to be installed through the new API endpoint.
-    // Checking for 'template' and 'composed_of' should catch them all.
-    // For the new v2 format, see https://github.com/elastic/elasticsearch/issues/53101
-    if (content.hasOwnProperty('template') || content.hasOwnProperty('composed_of')) {
-      templateAPIPath = '_index_template';
-    }
-
```

Comment:
This assumes that there will not be any legacy templates (https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-templates-v1.html) present in packages. Is this safe?